src/parachains/zombienet.rs

```diff
@@ -515,3 +515,336 @@ impl Source {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+
+	use super::*;
+	use anyhow::Result;
+
+	#[tokio::test]
+	async fn test_new_success() -> Result<()> {
+		//cache
+		let temp_dir = tempfile::tempdir().expect("Could not create temp dir");
+		let cache = PathBuf::from(temp_dir.path());
+
+		let zombienet = Zombienet::new(
+			cache.clone(),
+			"./tests/zombienet.toml",
+			Some(&"v1.7.0".to_string()),
```

Comment:
Might it make sense to use constants to reduce the repetition?

Reply:
yes, cleaner